library/src/main/java/com/timehop/stickyheadersrecyclerview/HeaderPositionCalculator.java

```diff
@@ -59,7 +59,7 @@ public Rect getHeaderBounds(RecyclerView recyclerView, View header, View firstVi
 
     if (firstHeader && isStickyHeaderBeingPushedOffscreen(recyclerView, header)) {
       View viewAfterNextHeader = getFirstViewUnobscuredByHeader(recyclerView, header);
-      int firstViewUnderHeaderPosition = recyclerView.getChildPosition(viewAfterNextHeader);
+      int firstViewUnderHeaderPosition = recyclerView.getChildAdapterPosition(viewAfterNextHeader);
```

Comment:
`isStickyHeaderBeingPushedOffscreen` checks for `NO_POSITION` and returns false in that case, so we'll never land in this if block with an invalid position.